### PR TITLE
Implement vehicle air placement

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Should work not fully tested.

### CODE REVIEW

File: src/OpenLoco/src/Windows/Vehicle.cpp [hunk 1]
```diff
@@ -3683,7 +3683,7 @@ namespace OpenLoco::Ui::Windows::Vehicle
             auto* airportObj = ObjectManager::get<AirportObject>(elStation->objectId());
             const auto& movementNode = airportObj->movementNodes[node];
             auto nodeOffset = Math::Vector::rotate(World::Pos2(movementNode.x, movementNode.y) - World::Pos2(16, 16), elStation->rotation()) + World::Pos2(16, 16);
-            auto nodeLoc = World::Pos3{ nodeOffset.x, nodeOffset.y, movementNode.y } + World::Pos3{ station->unk_tile_x, station->unk_tile_y, station->unk_tile_z };
+            auto nodeLoc = World::Pos3{ nodeOffset.x, nodeOffset.y, movementNode.z } + World::Pos3{ station->unk_tile_x, station->unk_tile_y, station->unk_tile_z };
```
[the remark this answers is not in the document]
Comment (AaronVanGeffen) [reply src/OpenLoco/src/Windows/Vehicle.cpp:3686]: I've not noticed anything odd about planes taxiing. Hoes does this manifest itself? Any kind of player-facing bug?

File: src/OpenLoco/src/GameCommands/Vehicles/VehiclePlaceAir.cpp [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+            if (!movementNode.hasFlags(AirportMovementNodeFlags::inFlight))
+            {
+                nodeLoc.z = 30 * 32;
+            }
+        }
+        return { nodeLoc };
+    }
+
+    // 0x004267BE
+    uint32_t vehiclePlaceAir(const VehicleAirPlacementArgs& args, uint8_t flags)
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/GameCommands/Vehicles/VehiclePlaceAir.cpp:74]:
```suggestion
    static uint32_t vehiclePlaceAir(const VehicleAirPlacementArgs& args, uint8_t flags)
```

File: src/OpenLoco/src/Windows/Vehicle.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (duncanspumpkin) [reply src/OpenLoco/src/Windows/Vehicle.cpp:3686]: I don't think the z component was being used by the window so probably nothing.

**Comment by AaronVanGeffen** [comment]
Trying to place an aircraft (cloned from an existing one), I repeatedly get the following error:

Screenshot 2024-04-05 at 21 28 08

**Review by AaronVanGeffen — APPROVED** [review]
Works fine for me with the latest changes. Nice to have the unk coordinates named as well :+1:

**Comment by duncanspumpkin** [comment]
Note: This is ready to go, doesn't desync replays but getAirportMovementNodeLoc needs to be deduplicated.

**Pull request merged**